src/api/java/baritone/api/utils/BlockOptionalMetaLookup.java

```diff
@@ -29,6 +29,10 @@
 
     private final BlockOptionalMeta[] boms;
 
+    public BlockOptionalMetaLookup() {
```

Comment:
Why is this necessary? Why can't the `...` constructor work with no arguments?

Reply:
because there are multiple ... constructors, so if you use no aruments your BlockOptionalMetaLookup() call gets ambiguous.